script.js

```diff
@@ -0,0 +1,310 @@
+    this.button.addEventListener('click', () => this.clearInput());
+    this.textArea.addEventListener('blur', () => {
+      this.textArea.focus();
+    });
+    window.addEventListener('blur', () => {
+      document.querySelectorAll('.activeKeyBoard').forEach((item) => item.classList.remove('activeKeyBoard'));
+    });
+    document.addEventListener('keydown', (e) => {
+      if (e.shiftKey && e.altKey) {
+        if (localStorage.getItem('language') === 'RU') {
+          this.engCase();
+        } else {
+          this.rusCase();
+        }
+      }
+    });
+  }
+
+  addVirtualKeys() {
+    for (let i = 0; i < 64; i += 1) {
+      const key = document.createElement('div');
+      key.classList.add('key');
+      key.classList.add(this.keyCode[i]);
+      this.wrapper.append(key);
+
+      const ruKey = document.createElement('div');
+      ruKey.classList.add('activeKeys');
+      this.wrapper.lastChild.append(ruKey);
+
+      const engKey = document.createElement('div');
+      engKey.classList.add('disableKeys');
+      this.wrapper.lastChild.append(engKey);
+
+      const ruKeyUpper = document.createElement('span');
+      ruKeyUpper.classList.add('off');
+      ruKeyUpper.innerHTML = this.ruUpperCase[i];
+      this.wrapper.lastChild.firstChild.append(ruKeyUpper);
+
+      const ruKeyLower = document.createElement('span');
+      ruKeyLower.classList.add('on');
+      ruKeyLower.innerHTML = this.ruLowerCase[i];
+      this.wrapper.lastChild.firstChild.append(ruKeyLower);
+
+      const engKeyUpper = document.createElement('span');
+      engKeyUpper.classList.add('off');
+      engKeyUpper.innerHTML = this.engUpperCase[i];
+      this.wrapper.lastChild.lastChild.append(engKeyUpper);
+
+      const engKeyLower = document.createElement('span');
+      engKeyLower.classList.add('on');
+      engKeyLower.innerHTML = this.engLowerCase[i];
+      this.wrapper.lastChild.lastChild.append(engKeyLower);
+
+      if (i === 14 || i === 29 || i === 42 || i === 55) {
+        key.classList.add('float-none');
+      }
+    }
+  }
+
+  checkKey(str) {
+    switch (str) {
+      case 'Enter':
+        this.textArea.value += '\n';
+        break;
+      case 'Tab':
+        this.textArea.value += '\t';
+        break;
+      case 'Backspace':
+        this.deleteSymbolPrev();
+        break;
+      case 'Delete':
+        this.deleteSymbolNext();
+        break;
+      case 'Del':
+        this.deleteSymbolNext();
+        break;
+      case 'ArrowLeft':
+        this.arrowMove('ArrowLeft');
+        break;
+      case 'ArrowRight':
+        this.arrowMove('ArrowRight');
+        break;
+      case 'ArrowUp':
+        this.arrowMove('ArrowUp');
+        break;
+      case 'ArrowDown':
+        this.arrowMove('ArrowDown');
+        break;
+      case 'CapsLock':
+        this.changeCase();
+        break;
+      default:
+    }
+  }
+
+  mouseClick(e) {
+    const target = e.target.closest('.key');
+    if (target == null) {
```

Comment:
You should use strict equality wherever possible.

Reply:
Fixed